settings.py

```diff
@@ -21,7 +21,6 @@ def path(*args):
 
 
 DEBUG = config('DEBUG', default=False, cast=bool)
-TEMPLATE_DEBUG = DEBUG
```

Comment:
Yeah. [Defaults to the value of `DEBUG`](https://docs.djangoproject.com/en/1.9/topics/templates/#django.template.backends.django.DjangoTemplates).